go.work

```diff
@@ -1,4 +1,4 @@
-go 1.23.0
+go 1.23.1
```

Comment:
is this needed? I thought when using toolchain, we could leave the `1.23.0` as baseline